distsql/select_result.go

```diff
@@ -142,7 +142,10 @@ func (r *selectResult) fetchResp(ctx context.Context) error {
 		for _, warning := range r.selectResp.Warnings {
 			sc.AppendWarning(terror.ClassTiKV.Synthesize(terror.ErrCode(warning.Code), warning.Msg))
 		}
-		r.feedback.Update(resultSubset.GetStartKey(), r.selectResp.OutputCounts)
+		if r.feedback != nil {
```

Comment:
and also when this will be `nil` :confused:~?

it seems table reader read multiple tables always has feedback info target their logical parent table

Reply:
I'm not sure the feedback is properly handled now when the KV range of a table reader crosses multiple partitions.
There is no test case broken, maybe we should revise here later.